libs/ssl-config/src/main/java/org/elasticsearch/common/ssl/StoredCertificate.java

```diff
@@ -57,10 +49,6 @@ public String getAlias() {
         return alias;
     }
 
-    public boolean hasPrivateKey() {
-        return hasPrivateKey;
-    }
-
     @Override
     public boolean equals(Object o) {
```

Comment:
Was it not possible to remove this instance of equals and hashCode?

Reply:
Good catch! I didn't finish the refactoring of `StoredCertificate` completely. I've removed the accessors and `equals&hashCode`